src/js/views/patients/sidebar/action/action-sidebar_views.js

```diff
@@ -68,6 +69,9 @@ const LayoutView = View.extend({
   onAttach() {
     animSidebar(this.el);
   },
+  onClose() {
+    Radio.trigger('user-activity', 'close:patient:action:sidebar');
```

Comment:
Doing a `this.triggerMethod()` doesn't work here. The flow app doesn't have access to it.